google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -357,8 +357,8 @@ def logging_prefix= logging_prefix
         ##
         # The bucket's storage class. This defines how objects in the bucket are
         # stored and determines the SLA and the cost of storage. Values include
-        # `MULTI_REGIONAL`, `REGIONAL`, `NEARLINE`, `COLDLINE`, and
-        # `DURABLE_REDUCED_AVAILABILITY`.
+        # `NEARLINE`, `COLDLINE`, and `DURABLE_REDUCED_AVAILABILITY`. `REGIONAL`
```

Comment:
This appears to be inconsistent with the other lists below: 

"Values include `NEARLINE`, `COLDLINE`, and `DURABLE_REDUCED_AVAILABILITY`."

vs

 "Accepted values include `:standard`, `:nearline`, and `:coldline`."